docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -1,42 +1,69 @@
+******************************
+Using Sawtooth on Ubuntu 16.04
+******************************
 
-.. _intro_to_sawtooth:
 
-*****************************************
-Getting Started with Hyperledger Sawtooth
-*****************************************
+This document guides you through the process of setting up Hyperledger Sawtooth
+for application development on Ubuntu, and then introduces basic Sawtooth
+concepts necessary for application development.
 
+After following the steps in this document, you will be able to perform the
+following tasks:
 
-Overview
-========
+* run Sawtooth
+* submit transactions
+* view blocks, transactions and state with the CLI
+* start and stop validators and transaction processors
 
-This tutorial shows you how to start a validator and transaction processor,
-then perform some basic operations, such as:
+You will then be prepared for the more advanced tutorials that guide you in
+performing app development tasks, such as implementing business logic with
+transaction families and writing clients which use Sawtooth's REST API.
 
-* Submitting transactions
-* Setting configuration settings using the config transaction family
-* Viewing blocks, transactions and state
 
-If you have not yet set your environment up, see :doc:`environment_setup` in
-the App Developer's Guide.
+Overview Sawtooth Components
+============================
 
-Prerequisites
--------------
+A running Sawtooth network consists of the following applications or processes:
 
-A supported app development environment is required. 
+.. image:: ../images/hyperledger_sawtooth_components.*
+   :width: 80%
+   :align: center
+   :alt: Sawtooth components
 
-See :doc:`environment_setup` for a guide to installing the app development
-environment.
+This diagram represents a simple network with just two validators and two
+transaction processors. The second validator's transaction processors are not
+depicted.
 
 
-Validator Start-up Process
-==========================
+Installation
+============
+
+Run the following commands from a terminal window, as root or with `sudo`:
+
+.. code-block:: console
+
+  $ echo "deb http://repo.sawtooth.me/ubuntu/0.8/stable xenial universe" >> /etc/apt/sources.list
+  $ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 6B58B1AC10FB5F63
+  $ apt-get update && apt-get install -y sawtooth
+
+
+Running As A Service
+====================
+
+When you install Sawtooth using apt-get, apt-get adds *systemd* units for
+the following components, which can then be started, stopped, and restarted
+using the *systemctl* command:
+
+* validator
+* transaction processors
+* rest_api
 
-.. caution::
+To run sawtooth components as a service, replace the manual commands in the next
+sections with a *systemctl* command.
```

Comment:
Do we need to address this for this version? What's the best idea to add something to this document for this sprint?